docs/deploying_serverless_applications.rst

```diff
@@ -53,4 +53,4 @@ Learn More
 -  `Installation <installation.rst>`__
 -  `Getting started with SAM and the SAM CLI <getting_started.rst>`__
 -  `Usage <usage.rst>`__
--  `Advanced <advanced_usage.rst>`__
\ No newline at end of file
+-  `Advanced <advanced_usage.rst>`__

```

Comment:
What is different here?

Reply:
It’s a white space at the end of the line that was removed

Reply:
Ahh cool. :)